frontend/src/HomePage/HomePage.jsx

```diff
@@ -194,6 +194,12 @@ class HomePage extends React.Component {
                             }
                             </>)
                           }
+                          {currentFolder.count == 0  && (
+                            <div>
+                              <img className = "mx-auto d-block" src ="assets/images/icons/empty-folder-svgrepo-com.svg" height="120px"/>
+                              <h3 className= "text-center">This folder is empty</h3>
```

Comment:
Can we use `span` tag with `text-body` as the class? Just to be consistent with other pages.

Reply:
Sure, will do that